3_javascript/javascript1/week2/homework javascript1 week2/candy.js

```diff
@@ -0,0 +1,49 @@
+//Candy helper optional //
+
+const boughtCandyPrices = [];
+
+const amountToSpend = Math.random() * 100;
+
+const candyPricePalette = [
+  {
+    type: "sweet",
+    price: 0.5,
+  },
+  {
+    type: "chocolate",
+    price: 0.07,
+  },
+  {
+    type: "toffee",
+    price: 1.1,
+  },
+  {
+    type: "chewing-gum",
+    price: 0.2,
+  },
+];
+function addCandy(candyType, weight) {
```

Comment:
What if we input some unknown candy type? 🍬

Reply:
Yes, it can altered by creating a new condition.